test/bridgeAdmin.ts

```diff
@@ -21,6 +21,27 @@ describe("Bridge Admin: ", function () {
   const singleTransactionLimit = convertToUnit(10, 18);
   const maxDailyTransactionLimit = convertToUnit(100, 18);
 
+  const functionregistry = [
```

Comment:
Should we add the rest of the functions listed here? https://github.com/VenusProtocol/token-bridge/blob/fix/VEN-2283/helpers/deploymentConfig.ts#L38C14-L38C33